NYPLAudiobookToolkit/Player/OpenAccessPlayer.swift

```diff
@@ -1,7 +1,15 @@
 import AVFoundation
 
-final class OpenAccessPlayer: NSObject, Player {
+class OpenAccessPlayer: NSObject, Player {
 
+    var errorDomain: String {
```

Comment:
why do we need this and the `taskCompleteNotification` computed properties? is it because they are part of the Player protocol?

Reply:
`OpenAccessPlayer` is the parent class of `OverdrivePlayer`, and those need to be changed